run-pr-tests

```diff
@@ -396,26 +390,60 @@ for WF in ${WORKFLOWS_FOR_VALGRIND_TEST//,/ }; do
   runTheMatrix.py --command '-n 10 --prefix "time valgrind --tool=memcheck --suppressions=$CMSSW_RELEASE_BASE/src/Utilities/ReleaseScripts/data/cms-valgrind-memcheck.supp --num-callers=20 --xml=yes --xml-file=valgrind.xml " ' -l $WF  
   popd
 done
-ls
-
 
 #
-# If auto post is selected...
+# test compilation with Clang
 #
-if [ "X$AUTO_POST_MESSAGE" = Xtrue ]; then
-  if $ALL_OK ; then
-     echo $ALL_OK
-     $WORKSPACE/cms-bot/report-pull-request-results TESTS_OK_PR --pr $PULL_REQUEST_NUMBER -c $LAST_COMMIT --pr-job-id $PULL_REQUEST_JOB_ID
-  elif [ "X$BUILD_OK" = Xfalse ]; then
-     echo $BUILD_OK
-     $WORKSPACE/cms-bot/report-pull-request-results PARSE_BUILD_FAIL --pr $PULL_REQUEST_NUMBER -c $LAST_COMMIT --pr-job-id $PULL_REQUEST_JOB_ID --unit-tests-file $WORKSPACE/build.log
-  elif [ "X$UNIT_TESTS_OK" = Xfalse ]; then
-     echo $UNIT_TESTS_OK
-     echo 'failure in unit tests'
-     $WORKSPACE/cms-bot/report-pull-request-results PARSE_UNIT_TESTS_FAIL --pr $PULL_REQUEST_NUMBER -c $LAST_COMMIT --pr-job-id $PULL_REQUEST_JOB_ID --unit-tests-file $WORKSPACE/unitTests.log
-  elif [ "X$RELVALS_OK" = Xfalse ]; then
-     pushd $WORKSPACE/
-     $WORKSPACE/cms-bot/report-pull-request-results PARSE_MATRIX_FAIL --pr $PULL_REQUEST_NUMBER -c $LAST_COMMIT --pr-job-id $PULL_REQUEST_JOB_ID --unit-tests-file $WORKSPACE/matrixTests.log
-     popd
+echo 'test clang compilation'
+
+#CONFIG_MAP_URL="https://raw.githubusercontent.com/cms-sw/cms-bot/master/config.map"
+CONFIG_MAP_URL='https://github.com/nclopezo/cms-bot/blob/test-compilation-clang-2b/config.map'
+
+NEED_CLANG_TEST=false
+if curl $CONFIG_MAP_URL | grep $RELEASE_QUEUE | grep PRS_TEST_CLANG= | grep SCRAM_ARCH=$ARCHITECTURE; then
+  NEED_CLANG_TEST=true
+fi
+
+if [ "X$TEST_CLANG_COMPILATION" = Xtrue -a $NEED_CLANG_TEST = true ]; then
+  $WORKSPACE/cms-bot/report-pull-request-results TESTS_RUNNING --pr $PULL_REQUEST_NUMBER -c $LAST_COMMIT --pr-job-id $PULL_REQUEST_JOB_ID --add-message "Testing Clang compilation" $DRY_RUN
+
+  #first, add the command to the log
+  echo "scram b vclean && scram build -k -j $(getconf _NPROCESSORS_ONLN) USER_CXXFLAGS='-fsyntax-only' COMPILER='llvm compile'" > $WORKSPACE/buildClang.log
```

Comment:
I had done it so it is executed at the end of all the other tests. But I just changed it to be at the beginning. and then  it does scram b vclean